examples/prompting/Basic_Classification.ipynb

```diff
@@ -0,0 +1,228 @@
+{
```

Comment:
<p>Can you use a system instruction for the preamble in this prompt?</p><p> </p><p>If you can/do, we can turn it into a standard few-shot / structured prompt too - e.g.</p><p> </p>

<pre class="ql-syntax" spellcheck="false">Topic: What can I do after highschool?
Comment: You should do a gap year!
Class: Neutral

...other examples...

Topic: {topic}
Comment: {comment}
Class: 
</pre><p> </p><p>This aligns with what we do in AI Studio (so what users see when exporting structured prompts), and is <em>slightly</em> more robust to accidental jailbreaks (e.g. the current prompt will accidentally jailbreak with a <code>"</code> in the comment or topic). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>